tests/samplers_tests/test_samplers.py

```diff
@@ -355,3 +356,24 @@ def objective(trial: Trial, base_value: float) -> float:
     # Non NaN objective value.
     study.optimize(lambda t: objective(t, 1), n_trials=1, catch=())
     assert int(study.best_value) == 1
+
+
+@parametrize_sampler
+def test_partial_fixed_sampling(sampler_class: Callable[[], BaseSampler]) -> None:
+
+    study = optuna.create_study(sampler=sampler_class())
+
+    def objective(trial: Trial) -> float:
+        x = trial.suggest_uniform("x", -1, 1)
+        y = trial.suggest_int("y", -1, 1)
+        return x + y
+
+    # First trial.
+    study.optimize(objective, n_trials=1)
+
+    # Second trial (`y` parameter is fixed as 0).
+    fixed_params = {"y": 0}
+    study.sampler = PartialFixedSampler(fixed_params, study.sampler)
+    study.optimize(objective, n_trials=1)
+    tmp_trial_params = study.trials[-1].params
+    assert tmp_trial_params["y"] == 0
```

Comment:
How about renaming it and using ''fixed_params''?
```suggestion
    trial_params = study.trials[-1].params
    assert trial_params["y"] == fixed_params["y"]
```